src/transformers/tokenization_utils_base.py

```diff
@@ -75,6 +76,9 @@
     if is_flax_available():
         import jax.numpy as jnp  # noqa: F401
 
+if is_protobuf_available():
+    from google.protobuf.message import DecodeError
```

Comment:
I think importing protobuff is hard see: 
https://github.com/huggingface/transformers/blob/3963a17762d45832fd13cd71d0a1daeeac4f4598/src/transformers/convert_slow_tokenizer.py#L33-L45

might need additional escapes ?

Reply:
i added en Error raise, but we only use it to catch the DecodeError and don't need `sentencepiece_model_pb2` , so I think it will be enough (see new commit)

Reply:
ah damn it actually fails on `is_protobuf_available` on CI https://app.circleci.com/pipelines/github/huggingface/transformers/99528/workflows/b5acf913-3fff-4b7b-9003-e7aff280c88d/jobs/1323973

Reply:
CI tests fixed! were unrelated